docs/user_manual/source/pipeline.rst

```diff
@@ -87,11 +83,25 @@ and zero stall on the data-side memory interface.
   |                       | 4 (target is a non-word-aligned      | EX stage and will cause a flush of the IF stage (including  |
   |                       | non-RVC instruction)                 | prefetch buffer) and ID stage.                              |
   +-----------------------+--------------------------------------+-------------------------------------------------------------+
-  | Instruction Fence     | 5                                    | The FENCE.I instruction as defined in 'Zifencei' of the     |
+  | Instruction Fence     | 5                                    | The FENCE[.I] instructions are defined in 'Zifencei' of the |
   |                       |                                      | RISC-V specification. Internally it is implemented as a     |
   |                       | 6 (target is a non-word-aligned      | jump to the instruction following the fence. The jump       |
   |                       | non-RVC instruction)                 | performs the required flushing as described above.          |
+  |                       |                                      | A FENCE.I instruction will not complete until the external  |
+  |                       |                                      | handshake has been completed.                               |
   +-----------------------+--------------------------------------+-------------------------------------------------------------+
+  | Zba, Zbb, Zbc, Zbs    | 1                                    | All instructions from Zba, Zbb, Zbc, Zbs take 1 cycle.      |
+  +-----------------------+--------------------------------------+-------------------------------------------------------------+
+  | Zcmt                  | 2                                    | Tablejumps take 2 cycles.                                   |
+  +-----------------------+--------------------------------------+-------------------------------------------------------------+
+  | Zcmp                  | 2 - 18                               | The number of cycles depend on the number of registers      |
+  |                       |                                      | saved or restored by the instructions.                      |
+  +-----------------------+--------------------------------------+-------------------------------------------------------------+
+  | Zca, Zcb              | 1                                    | Instructions from Zca and Zcv take 1 cycle.                 |
+  +-----------------------+--------------------------------------+-------------------------------------------------------------+
+  | WFI, WFE              | 1 -                                  | Instructions causing sleep will not retire until wakeup.    |
```

Comment:
Is it really 1 - or is it for example 2 - ? Under what scenario can '1' happen?

Reply:
It should indeed be 2 -, unless we also specify behavior under debug mode which would simply be '1'. Not including debug mode for now.